scss/variables/breakpoints.js

```diff
@@ -0,0 +1,32 @@
+( function( root, factory ) {
+	if ( typeof define === "function" && define.amd ) {
+		define( [ "./chassis" ], factory );
+	} else if ( typeof exports === "object" ) {
+		module.exports = factory( require( "./chassis" ) );
+	} else {
+		root.chassis = factory( root.chassis );
+	}
+}( this, function( chassis ) {
+
+chassis.breakpoints = {
+
+	"viewport-xs": {
```

Comment:
I suggest using a `-min` or `-max` suffix for these variable names to clarify whether they're the start or the end of a viewport size tier.